jsapp/js/components/ccpmDocxReport201130.es6

```diff
@@ -1,35 +1,63 @@
-import {Document} from "docx";
-import dataset from '../ccpmDataset';
+import {Document, TabStopPosition, TabStopType} from "docx";
+import dataset, {ccpm_parseNumber} from '../ccpmDataset';
 import {ccpm_getStatusLabel, ccpm_getStatusColor} from '../ccpmReport';
-import {TextRun, Paragraph, ImageRun, SectionType, Table, TableRow, TableCell, WidthType} from 'docx';
+import {TextRun, Paragraph, ImageRun, SectionType, Table, TableRow, TableCell, WidthType, SymbolRun} from 'docx';
 
-const getTable2 = (data, length, border = false, marginBottom = 150, leftMargin = 40) => {
+const getTable2 = (data, length, border = false, marginBottom = 150, leftMargin = 40, top = 0, size = 100) => {
   if (!data) return;
   const columWidth = new Array(length);
   return new Table({
       columnWidths: columWidth.map(w => 9000/columWidth.length),
-      width:{size: 100, type:WidthType.PERCENTAGE},
+      width:{size, type:WidthType.PERCENTAGE},
       margins:{
         top: 150,
         bottom: marginBottom,
         left: leftMargin,
-        right: 40
+        right: 40,
+        marginUnitType: 'dxa'
       },
       borders: border,
       rows: [...data.map(t=>new TableRow({
         children: t.map(tt => new TableCell({
+          margins: {top},
           children: [tt],
           columnSpan: 2
         }))
       }))]
     })
 }
 
+const getP12Question = (parentState) => {
+  const {p12Result} = parentState;
+  const no = p12Result.filter(res => res['Partner_Survey_GROUP/Partner_Inform_Strategy_GROUP/P_IS02'] === 'no');
```

Comment:
Do you think there is a way to identify P_IS02 without specifying the whole question path? Just wondering in case future surveys are named or grouped differently, so the question finding will fail.